test/syscalls/linux/exec_binary.cc

```diff
@@ -914,6 +923,64 @@ TEST(ElfTest, ELFInterpreter) {
               })));
 }
 
+// Prelinked dynamically linked binary with an ELF interpreter.
```

Comment:
Perhaps just call this a shared object with a specific load address? That's really what is special here.